### PR TITLE
[14.0] [MIG] account_(bank_)statement_import_online(_ponto)

### PR DESCRIPTION
**Pull request by NL66278**
Migrate (and rename) both account_bank_statement_import_online as well as account_(bank_)statement_import_online_ponto.

The ... _ponto is the only online provider implemented in 13.0, and obviously it can not be migrated without the base module.

### CODE REVIEW

**Comment by alexis-via** [comment]
Requires PR https://github.com/OCA/web/pull/1934

**Comment by alexis-via** [comment]
I can't even install account_statement_import_online of this PR on a v14 database:
- data/account_bank_statement_import_online.xml : the module name in model_id must be renamed
- views/account_journal.xml: there is an inherit of account.view_account_bank_journal_form which doesn't exist in v14. 
- in the same file, there is an inherit of account_bank_statement_import.journal_dashboard_view_inherit => module name is wrong in v14
@NL66278 What happened ? Maybe you pushed the wrong code?

**Comment by NL66278** [comment]
@alexis-via My apologies, should have made clear that this was still WIP. Thanx for pointing out the missing parts anyway!

**Comment by alexis-via** [comment]
There is still several things to update:
- fix a view that prevent the module from being installed
- in online_bank_statement_provider.py: use of journal.sequence_id => sequence_id field doesn't exist any more in v14

**Comment by NL66278** [comment]
@alexis-via Should be good now. Hope to have a customer account for real live testing in a few days.

**Comment by alexis-via** [comment]
I improved the PR a bit in this PR: https://github.com/NL66278/bank-statement-import/pull/8

**Comment by ernestotejeda** [comment]
@NL66278 , since I need this module and there is also a mix of migration commits, I will proceed to supersede this PR. Is it OK for you?

**Comment by NL66278** [comment]
@ernestotejeda Please elaborate? I am fixing up a last test that is failing (should not effect actual function). If you have any extra change, you could submit it as a PR to my branch?

**Comment by ernestotejeda** [comment]
@NL66278  If you are working on the PR, it's OK :). Anyway I suggest, please, that you leave only two commits for each module, one for pre-commit related changes and another one for migration related changes.

**Review by ernestotejeda — COMMENTED** [review]
Code review: :+1:

**Review by ernestotejeda — COMMENTED** [review]
@NL66278 I don't see the pre-commit commit.

**Comment by pedrobaeza** [comment]
@NL66278 Please avoid merge commits in any PR, and squash your migrations commits according guidelines.

File: account_statement_import_online/README.rst [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure online bank statements provider:
+
+#. Go to *Invoicing > Configuration > Bank Accounts*
```
Comment (ernestotejeda) [inline account_statement_import_online/README.rst:40]: This menu no longer exist.

**Review by ernestotejeda — COMMENTED** [review]
Code review: :+1: 
I approve the changes made to the code, but I would have liked more that each commit had changes related to a single module. Anyway LGTM.

**Review by ernestotejeda — APPROVED** [review]

**Pull request merged**